app/src/main/java/com/keylesspalace/tusky/di/NetworkModule.kt

```diff
@@ -77,7 +77,7 @@ class NetworkModule {
                 .apply {
                     addInterceptor(InstanceSwitchAuthInterceptor(accountManager))
                     if (BuildConfig.DEBUG) {
-                        addInterceptor(HttpLoggingInterceptor().setLevel(HttpLoggingInterceptor.Level.BASIC))
+                        addInterceptor(HttpLoggingInterceptor().setLevel(HttpLoggingInterceptor.Level.BODY))
```

Comment:
Is this part of the counter functionality?

Reply:
Is not, good spot, I'll remove it. Apologies, taken this branch on from a previous dev and it was mixed in with some other features. Thought I'd untangled it all, apparently missed the odd shred.